Bottom: fd4d905c3efcf81f80b22150a19fb5630881d10e
Top:    a246ad5c6354a16112a5a45be4e303af83ee20dd
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-11-10 23:07:44 -0500

Fix outline drawing to only apply on copper layers

This matches the old comments in the code, and fixes (removes) outline
drawing on the solder mask prints.

This partially reverts commit 294494e56695251e63c0da9f243d6f59447f1c9f
"Fix PS rect/outline logic", which changed the logic, and
commit d2458383a60b670ad1c42163ac24fcc63fcab457 where I changed
the comment to match the code.


---

diff --git a/src/hid/ps/ps.c b/src/hid/ps/ps.c
index 31eea9d..5587347 100644
--- a/src/hid/ps/ps.c
+++ b/src/hid/ps/ps.c
@@ -992,10 +992,11 @@ ps_set_layer (const char *name, int group, int empty)
     fprintf (global.f, "gsave tx ty translate 1 -1 scale 0 0 moveto ( ) show grestore newpath /ty ty ts sub def\n");
 #endif
 
-  /* If we're printing a layer other than the outline layer, and
-     we want to "print outlines", and we have an outline layer,
+  /* If we're printing a copper layer other than the outline layer,
+     and we want to "print outlines", and we have an outline layer,
      print the outline layer on this layer also.  */
   if (global.outline &&
+      global.is_copper &&
       global.outline_layer != NULL &&
       global.outline_layer != PCB->Data->Layer+idx &&
       strcmp (name, "outline") != 0 &&
